models/Note.js

```diff
@@ -25,7 +25,13 @@ Note.init(
     created_at: {
         type: DataTypes.DATE,
         allowNull: false,
-        defaultValue: DataTypes.NOW
+        defaultValue: sequelize.literal("CURRENT_TIMESTAMP")
```

Comment:
im pretty sure that we just specify timestamp:true in the definition portion

Reply:
you can see an example using timestamps here https://github.com/IanSSenne/mvc-challenge/blob/main/models/User.js#L36